frontend/fingertips-frontend/components/forms/SearchForm/index.tsx

```diff
@@ -10,34 +17,57 @@ const StyledInputField = styled(InputField)(
   spacing.withWhiteSpace({ marginBottom: 6 })
 );
 
-export const SearchForm = ({ indicator }: { indicator: string }) => {
-  const initialState: SearchFormState = {
-    indicator: '',
-    message: null,
-    errors: {},
-  };
-  const [_state, formAction] = useActionState(searchIndicator, initialState);
+export const SearchForm = ({
+  searchFormState,
+}: {
+  searchFormState: SearchFormState;
+}) => {
+  const [state, formAction] = useActionState(searchIndicator, searchFormState);
 
   return (
-    <>
-      <H1>Search</H1>
+    <main>
       <form action={formAction}>
+        {state.message && (
+          <ErrorSummary
+            description="At least one of the following fields must be populated:"
+            errors={[
+              {
+                targetName: 'indicator',
+                text: 'Indicator field',
```

Comment:
My expectation (and I think this is backed up by [the Gov.UK frontend guidance](https://design-system.service.gov.uk/components/error-summary/#linking-from-the-error-summary-to-each-answer)) is that clicking the link in the error summary should scroll the relevant field (i.e. the indicator field) into view and bring it into focus. This isn't working for me.

Reply:
I don't think it says that it sets focus to the field, I think it should just scroll the offending field into view (if it isn't).

Reply:
I agree it's not clear about focusing the field (it just says to "link to" the field), but the behaviour of the examples on the page I linked is to focus the linked field, as well as scrolling to it.

Regardless of that, scrolling to the field isn't working for me on the page as it is now. I don't get taken to the error summary when it is shown, and clicking the link to the indicator field doesn't appear to do anything.